kalibro_client/base.py

```diff
@@ -103,6 +103,15 @@ def array_to_objects_array(cls, array):
     def exists(cls, id):
         return cls.request(':id/exists', params={'id': id}, method='get')['exists']
 
+    def delete(self):
+        if not isinstance( self.id, ( int, long ) ):
```

Comment:
Don't you think it's better to remove white spaces before closing and after opening parenthesis?